src/CommunityToolkit.Maui.Core/Views/Popup/PopupView.macios.cs

```diff
@@ -2,40 +2,74 @@
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 using System.Linq;
-using CommunityToolkit.Maui.Extensions.Internals;
 using CoreAnimation;
 using CoreGraphics;
-using CoreGraphics;
-using UIKit;
 using UIKit;
 
-namespace CommunityToolkit.Maui.Views.Popup;
+namespace CommunityToolkit.Maui.Core.Views;
 
-class PopupViewVisualOptions
+/// <summary>
+/// Visual Options for <see cref="PopupView"/>
+/// </summary>
+public class PopupViewVisualOptions
```

Comment:
why SnackbarOptions have its own file, but this one is here?

Reply:
That's fair. I'll move it 👍

Reply:
Done ✅